torch/_dynamo/utils.py

```diff
@@ -398,6 +398,10 @@ def clone_tensor(x):
 
 def clone_input(x):
     """copy while preserving strides"""
+    # TODO: this is questionable
+    if isinstance(x, torch._subclasses.FakeTensor):
+        # this func fails on fake tensors in __torch_dispatch__
+        return x
```

Comment:
We had umbrage with this in the source PR, did we ever get to talk about what a better course might be?